install.sh

```diff
@@ -24,6 +24,7 @@ function display_help()
 #  echo "    [--cuda] build library for cuda backend"
   echo "    [--cpu_ref_lib] specify libary to use for cpu reference code in testing (blis or lapack)"
   echo "    [--hip-clang] build library for amdgpu backend using hip-clang"
+  echo "    [-n|--no_tensile] build subset of library that doesn't require tensile (testing)"
```

Comment:
The `(testing)` remark at the end should probably be rewritten as `(beta)`, `(under development)` or `(experimental)`, unless you really are referring to Tensile or other "testing".

Reply:
Oh, and `Tensile` is usually capitalized.

Reply:
I had already merged, I'll adjust these later in another PR if okay